pkg/operator/starter.go

```diff
@@ -238,14 +238,22 @@ var deploymentConfigMaps = []revision.RevisionResource{
 	{Name: "serviceaccount-ca"},
 	{Name: "service-ca"},
 	{Name: "recycler-config"},
+
+	// created by certrotation controller
+	{Name: "kcm-localhost-serving-ca"},
+	{Name: "cpc-localhost-serving-ca"},
 }
 
 // deploymentSecrets is a list of secrets that are directly copied for the current values.  A different actor/controller modifies these.
 var deploymentSecrets = []revision.RevisionResource{
 	{Name: "service-account-private-key"},
 
+	// created by certrotation controller
+	{Name: "kcm-localhost-serving-cert"},
+	{Name: "cpc-localhost-serving-cert"},
```

Comment:
we may get an err/warnings for a while in the operator before the certrorationcontroller creates the resources. This might take a few seconds and generate 10s of errors. Is that okay?

```
E0421 17:51:38.100472       1 base_controller.go:270] "InstallerController" controller failed to sync "key", err: missing required resources: [secrets: csr-signer,kube-controller-manager-client-cert-key, configmaps: cpc-localhost-serving-ca-7,kcm-localhost-serving-ca-7, secrets: cpc-localhost-serving-cert-7,kcm-localhost-serving-cert-7,localhost-recovery-client-token-7,service-account-private-key-7]
I0421 17:51:38.261844       1 event.go:282] Event(v1.ObjectReference{Kind:"Deployment", Namespace:"openshift-kube-controller-manager-operator", Name:"kube-controller-manager-operator", UID:"19fa4dd8-0702-4c69-8768-6a6748af6258", APIVersion:"apps/v1", ResourceVersion:"", FieldPath:""}): type: 'Warning' reason: 'RequiredInstallerResourcesMissing' secrets: csr-signer,kube-controller-manager-client-cert-key, configmaps: cpc-localhost-serving-ca-7,kcm-localhost-serving-ca-7, secrets: cpc-localhost-serving-cert-7,kcm-localhost-serving-cert-7,localhost-recovery-client-token-7,service-account-private-key-7
```